### ISSUE TITLE
A lot of packages cannot be found

### ISSUE THREAD

**Issue by nowave7** [storeman-developers/harbour-storeman#162]
I just updated the SFOS to the latest stable release (4.3.0.15), and since then I'm having trouble installing applications through storeman. Now, I'm not entirely sure if this is related, since I haven't been installing or updating any apps in some time now, probably weeks, so it might be purely coincidental.
In any case, I am able to install some, and unable to install other applications. Doesn't seem to be a rule to it.
The only error reported in the UI is:
"An error occured
Couldn't find package"
Starting the app from terminal yields the following:
`[defaultuser@XperiaXA2 Downloads]$ /usr/bin/harbour-storeman
[D] unknown:0 - Using Wayland-EGL
[W] unknown:126 - file:///usr/share/harbour-storeman/qml/harbour-storeman.qml:126:30: QML MainPage: Binding loop detected for property "status"
[W] unknown:126 - file:///usr/share/harbour-storeman/qml/harbour-storeman.qml:126:30: QML MainPage: Binding loop detected for property "status"
[W] unknown:0 - OrnAppsModel(0x4b365d8) Model is already fetching data
[W] unknown:0 - libpng warning: iCCP: known incorrect sRGB profile
[C] unknown:0 - Could not read "/home/.zypp-cache/solv/openrepos-planetos_store/solv"
[C] unknown:0 - Could not read "/home/.zypp-cache/solv/openrepos-karin_zhao/solv"
[C] unknown:0 - Could not read "/home/.zypp-cache/solv/openrepos-black_sheep_dev/solv"
[C] unknown:0 - Could not read "/home/.zypp-cache/solv/openrepos-Rikudou_Sennin/solv"
`
Though all the errors were present upon just starting the storeman, nothin new comes when trying to install an app.

Any thoughts where we could get more insight into what the problem might be?
Thanks!

**Comment by Olf0** [comment]
Which Storeman version are you using?  See "About" in pulley menu.

**Comment by Olf0** [comment]
Please execute in the terminal:
```
devel-su
pkcon refresh
zypper refresh
```
If you do not have `zypper` installed, the last line will fail and that is O.K. (i.e., do not install `zypper` to execute it).

**Comment by nowave7** [comment]
Hey Olf0, I've did the pkcon refresh, multiple times as well, but still have some issues installing some packages. One more interesting thing is that there are also some apps that don't even have the install option just refresh cache and reload.

**Comment by Olf0** [comment]
> And you should consider to update to a recent version of Storeman via Storeman Installer: https://openrepos.net/content/osetr/storeman-installer

O.K., [you seem to have tried that and Storeman Installer did not start](https://openrepos.net/comment/40157#comment-40157), correct?

Then download the latest Storeman 0.2.11 from https://build.sailfishos.org/package/show/home:mentaljam/harbour-storeman
I.e. for your Xperia XA2 (armv7hl) on SFOS 4.3.0: https://repo.sailfishos.org/obs/home:/mentaljam/4.3.0.12_armv7hl/armv7hl/harbour-storeman-0.2.11+sfos4.2.20210918134855.2.g1945da7-1.34.1.jolla.armv7hl.rpm
Install it via filemanager or `pkcon install-local harbour-storeman-0.2.11+sfos4.2.20210918134855.2.g1945da7-1.34.1.jolla.armv7hl.rpm`

**Comment by nowave7** [comment]
Yes, I've had issues installing it through the installer package. So I ended up installing it manually from the command line like you explained.
When it comes to the package I referenced in my previous post, the problem turned out to be something else, namely a missing dependency for the theme, which was solved by installing the Clockwork, sorry for the unintentional confusion.
There are, however, still some packages that just report package not found when trying to install them.
It looks to me that, once the repository that holds the package has been added, the `pkcon refresh` needs to be run even though the GUI forntend also does the refresh, and sometimes not just once... Not sure why that is. I'm not that familiar with this package manager (apt-get kind of guy).
In any case, I have a workaround for this, so it's not a deal breaker, and being that the development of the storeman itself is now put into question, I guess we can close this issue.
